diff --git a/node_modules/mdsvex/dist/main-CkT3KbbK.js b/node_modules/mdsvex/dist/main-CkT3KbbK.js
index 0ef0100..39a9ac2 100644
--- a/node_modules/mdsvex/dist/main-CkT3KbbK.js
+++ b/node_modules/mdsvex/dist/main-CkT3KbbK.js
@@ -24327,7 +24327,7 @@ function transform(
 ;
 
 	processor.add_layouts = (layout, layout_mode) => {
-		processor.use(transform_hast, { layout: layout, layout_mode });
+		processor(transform_hast, { layout: layout, layout_mode });
 	};
 
 	return processor;
